reana_db/models.py

```diff
@@ -60,6 +60,7 @@ class WorkflowStatus(enum.Enum):
     running = 1
     finished = 2
     failed = 3
+    deleted = 4
```

Comment:
Can you please add `stopped` already? (according to https://github.com/reanahub/reana-client/issues/192#issuecomment-437003151 we should be fine with those status for now). And also rebase.

Reply:
I think it's better to add it in the PR group for `workflow stop`, if it is part of this PR, it won't show for example that the `workflow stop` adds something to the DB.

Reply:
Agree 👍 can you rebase?